doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -27,12 +98,10 @@ Cockpit is an interactive Linux server administrative interface. You can perform
 Cockpit must be pre-configured and running on the virtual machine. 
 ====
 
-As you can see in the following screen capture of the Self Service user interface, the cockpit button is available next to the HTML 5 console button for a virtual machine instance in the *My Services* tab. 
+The option to open the Cockpit console for a virtual machine is available in the image:ssui-access-button.png[Access Button] button, which also houses the option to open HTML5 Console, in the *Resources* section inside the *Services* tab. 
```

Comment:
[WIP] I've made this edit, but please double-check that this looks OK on your next pass of the chapter.